mcs/class/System.Net.Http/System.Net.Http.Headers/HttpHeaders.cs

```diff
@@ -193,7 +193,7 @@ bool AddInternal (string name, IEnumerable<string> values, HeaderInfo headerInfo
 							continue;
 						}
 
-						throw new FormatException ();
+						throw new FormatException ($"{name}, {value}");
```

Comment:
Could you please add a better message, e.g. `Couldn't parse header '{name}' with value '{value}'`?